Open Judge System/OJS.Common/Extensions/DirectoryHelpers.cs

```diff
@@ -30,5 +46,32 @@ public static void SafeDeleteDirectory(string path, bool recursive = false)
                 Directory.Delete(path, recursive);
             }
         }
+
+        public static void DeleteExecutionStrategiesWorkingDirectories()
```

Comment:
`DeleteExecutionStrategyWorkingDirectories` - by convention we tend to pluralize only last word